db/fixtures/customization_templates.yml

```diff
@@ -275,6 +275,116 @@
   :type: CustomizationTemplateSysprep
   :system: true
 
+- :name: Windows 2012
+  :description: This is a template for Windows 2012 R2
```

Comment:
Let's follow the existing 2008 Sysprep's name/description text for this 2012 version.

https://github.com/ManageIQ/manageiq/blob/master/db/fixtures/customization_templates.yml#L206-L207

```yaml
- :name: Windows Server 2008 R2
  :description: Windows Server 2008 R2 Enterprise x64
```